subsys/spm/Kconfig

```diff
@@ -171,6 +171,15 @@ config SPM_NRF_WDT_NS
 	bool "WDT is Non-Secure"
 	default n
 
+config SPM_NRF_DPPIC_NS
+	bool "DPPIC is Non-Secure"
+	default y
+
+config SPM_NRF_DPPIC_PERM_MASK
+	hex "Set DPPIC PERM mask for Non-secure"
+	default 0x0000FFFF
```

Comment:
Can you explain in a help text, how this option is used, and how the default value is retrieved?